docs/UserGuide.md

```diff
@@ -3,11 +3,13 @@
 title: User Guide
 ---
 
-# Your guide to QuickCache
+# Welcome to QuickCache User Guide
 
 **QuickCache** is a desktop app for managing flashcards, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
 
-If you are a student who can type fast and loves organizing your study materials, QuickCache can get your flashcard management tasks done faster than any traditional GUI appplication.
+If you are a student who can type fast and loves organizing your study materials, **QuickCache** can get your flashcard management tasks done faster than any traditional GUI appplication.
+
+The **QuickCache User Guide** helps you get started with using **QuickCache**. This user guide aims to walk you through the robust features that **QuickCache** has to offer and take you from zero to hero. Don't like reading? A summary is available at the end for your reference.
```

Comment:
Nice. I think can consider adding a link to the words "summary" that will link to the command summary section